openfisca_core/scripts/run_test.py

```diff
@@ -47,7 +47,10 @@ def main():
         if len(installed_country_packages) > 1:
             print('WARNING: Several country packages detected : `{}`. Using `{}` by default. To use another package, please use the --country_package option.'.format(', '.join(installed_country_packages), country_package_name))
 
-    tax_benefit_system = country_package.CountryTaxBenefitSystem()
+    try:
+        tax_benefit_system = country_package.CountryTaxBenefitSystem()
+    except AttributeError:
+        tax_benefit_system = country_package.country_tax_benefit_system
```

Comment:
Tell me if I'm wrong but I don't see `country_tax_benefit_system` in https://github.com/openfisca/openfisca-france-data/blob/master/openfisca_france_data/__init__.py, but `france_data_tax_benefit_system` 🤔